### PR TITLE
DOCSP-17146 completed minimum distance sentence under aggregation page

### PR DESCRIPTION
**Pull request by ianf-mongodb**

### CODE REVIEW

File: source/reference/operator/aggregation/geoNear.txt [hunk 1]
```diff
@@ -278,7 +278,9 @@ Minimum Distance
 
 The following example uses the option ``minDistance`` to specify
 the minimum distance from the center point that the documents can be.
-The following aggregation finds all documents that 
+The following aggregation finds all documents with a location at least
+2 meters from the center [ -73.99279 , 40.719296 ] and ``category`` 
+equal to ``Parks``.
```
Comment (andf-mongodb) [inline source/reference/operator/aggregation/geoNear.txt:283]: Hi Ian --

Tiny nit: let's render the coordinates ( `[ -73.99279 , 40.719296 ]` ) as monospace also as with the previous example intro.

**Review by andf-mongodb — COMMENTED** [review]
Hi Ian, small nit regarding monospace.

File: source/reference/operator/aggregation/geoNear.txt (hunk 1 above)
Comment (ianf-mongodb) [reply source/reference/operator/aggregation/geoNear.txt:283]: Hi Andrew, added monospace per request. Thanks!

Comment (andf-mongodb) [reply source/reference/operator/aggregation/geoNear.txt:283]: Thanks! Looks good!

**Review by andf-mongodb — APPROVED** [review]
LGTM!

**Pull request merged**